### PR TITLE
3078 refresh on copy

### PR DESCRIPTION
**Pull request by cakekoa**
# What does this PR do?

Fixes part of #3078 by refreshing the OTP when the command is copied.

- Refreshes the command if the copy button is pressed
- Refreshes the command if `ctrl+c` is pressed

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/CommandDisplay.js [hunk 1]
```diff
@@ -26,6 +26,22 @@ export default function commandDisplay(props) {
...
+      props.onCopy();
+    }
+    if ((event.ctrlKey || event.metaKey) && charCode === 'a') {
+      event.preventDefault();
+      let codeElement = event.target.querySelector('code');
+      let selection = window.getSelection();
+      let range = document.createRange();
+      range.selectNode(codeElement);
+      selection.addRange(range);
+    }
```
Comment (shreyamalviya) [inline monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/CommandDisplay.js:41]: This change is so that instead of everything on the webpage getting selected on ctrl+A, only the command does?
1. Not sure when/why but it isn't working each time.
1. I feel this is out of scope. Let's focus on copying.

**Review by shreyamalviya — CHANGES_REQUESTED** [review]
OTP doesn't change if I right-click -> copy. We should take care of that too.

File: monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/CommandDisplay.js (hunk 1 above)
Comment (ilija-lazoroski) [reply monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/CommandDisplay.js:41]: +1

**Review by shreyamalviya — COMMENTED** [review]
I don't like this a lot. There's no way for the user to know that copying the command will refresh the OTP. If they want to refresh it, they can do it using the refresh button, and it'll refresh on its own on expiry once the timer is in place, which the user can see clearly too. I say we get rid of this.

**Comment by ilija-lazoroski** [comment]
> I don't like this a lot. There's no way for the user to know that copying the command will refresh the OTP. If they want to refresh it, they can do it using the refresh button, and it'll refresh on its own on expiry once the timer is in place, which the user can see clearly too. I say we get rid of this.

I agree with @shreyamalviya on this. I don't see any purpose to refresh it on copy except for some in-depth security.

**Comment by mssalvatore** [comment]
After some discussion, we decided to leave the "refresh on copy" functionality. As far as the UI is concerned, once the user has copied the command, the OTP can be considered to be used.
